test/Ocelot.AcceptanceTests/Steps.cs

```diff
@@ -381,7 +459,7 @@ public void WhenIGetUrlOnTheApiGatewayWaitingForTheResponseToBeOk(string url)
         {
             try
             {
-                _response = _ocelotClient.GetAsync(url).Result;
+                _response = _ocelotClient.GetAsync(url).GetAwaiter().GetResult();
```

Comment:
Did you try to convert the method to `async`? And after that ->
```suggestion
                _response = await _ocelotClient.GetAsync(url);
```

Reply:
I tried but I couldn't. I retried and now I could came up with the async version for `Until` which is, of course, called `UntilAsync`. This way we got rid some `GetAwaiter().GetResult()`.

Reply:
Seems this is special case of the test. So, the test must wait synchronously...
Not an issue!